plugins/autocomplete/plugin.js

```diff
@@ -1070,4 +1079,14 @@
 	CKEDITOR.plugins.autocomplete = Autocomplete;
 	Autocomplete.view = View;
 	Autocomplete.model = Model;
+
+	/**
+	 * The autocomplete key codes used to finish autocompletion with selected view item.
+	 * This setting will set completing key codes for each autocomplete plugin respectively.
+	 * To change completing key codes individually use {@link CKEDITOR.plugins.autocomplete#completingKeyCodes} plugin property.
+	 *
+	 * @cfg {Number[]} [autocomplete_completingKeyCodes=[9, 13] (9 = tab, 13 = enter, empty array = disabled)]
```

Comment:
Missing `@since` tag.

Reply:
How about simplifying it to `autocomplete_commitKeystroke`? Somehow better to me, you could do a quick voting on Slack or sth 🙂. And in that case it could be either a `Number` (convienient) or `Number[]`.

Reply:
Looks like `autocomplete_commitKeystroke` won 😄